tests/testAPI.sh

```diff
@@ -21,6 +21,29 @@ KRUIZE_URL="http://localhost"
 KRUIZE_PORT="31313"
 AUTH_TOKEN=""
 
+function usage() {
+	echo "Usage: $0 [-u kruize_url] [-t kruize_port] [-a auth_token]"
```

Comment:
Kruize port should be -p

Reply:
Also please add `echo` above this line so that a new line might be printed

Reply:
Done